packages/edit-post/src/index.js

```diff
@@ -185,15 +134,13 @@ export function initializeEditor(
 	window.addEventListener( 'dragover', ( e ) => e.preventDefault(), false );
 	window.addEventListener( 'drop', ( e ) => e.preventDefault(), false );
 
-	render(
+	createRoot( target ).render(
```

Comment:
Might be worth returning the `root` in `initializeEditor` as well. So that we can allow users to unmount the tree as `unmountComponentAtNode` is replaced.